docs/basics/101-130-yodaproject.rst

```diff
@@ -0,0 +1,564 @@
+widely available, among many other sources as
+`Github gists <https://gist.github.com/netj/8836201>`_.
+
+To start your analysis project and comply to the YODA principles, you set up
+an independent data dataset with your project's raw data. For this, create a
+new dataset outside of ``DataLad-101``:
+
+.. runrecord:: _examples/DL-101-129-101
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   # make sure to move outside of DataLad-101!
+   $ cd ../
+   $ datalad create iris_data
+
+Inside of this newly created dataset, get the data. It is publicly
+available from a
+`Github Gist <https://gist.github.com/netj/8836201>`_,
+and we can use :command:`datalad download-url` to get it:
+
+.. findoutmore:: What are Github Gists?
+
+   Github Gists are a particular service offered by Github that allow users
+   to share pieces of code snippets and other short/small standalone
+   information. Find out more on Gists
+   `here <https://help.github.com/en/github/writing-on-github/creating-gists#about-gists>`__.
+
+
+.. runrecord:: _examples/DL-101-130-102
+   :workdir: dl-101
+   :language: console
+
+   $ cd iris_data
+   $ datalad download-url https://gist.githubusercontent.com/netj/8836201/raw/6f9306ad21398ea43cba4f7d537619d0e07d5ae3/iris.csv
+
+.. todo::
+
+   or do we rather want to install/clone the dataset?
+
+This downloaded the iris dataset as a comma-seperated (``.csv``) file, and,
+importantly, recorded where it was obtained from. "Nice, this way I have
+sufficient provenance capture for my input dataset!" you think as you
+mentally tick of YODA principle 2 for your input data.
+
+Now that you have this raw dataset, it is time to create an analysis for your
+midterm project.
+You start by creating an analysis dataset. Let's do it as a subdataset
+of ``DataLad-101``. Remember to specify the ``--dataset`` option within
+:command:`datalad create` to link it as a subdataset:
+
+.. runrecord:: _examples/DL-101-130-103
+   :language: console
+   :workdir: dl-101/iris_data
+
+   # go back into DataLad-101
+   $ cd ../DataLad-101
+   $ datalad create --dataset . midterm_project
+
+.. index:: ! datalad command; datalad subdatasets
+
+The :command:`datalad subdatasets` command can report on which subdatasets exist for
+``DataLad-101``. This helps you verify that the command succeeded and the
+dataset was indeed linked as a subdataset to ``DataLad-101``:
+
+.. runrecord:: _examples/DL-101-130-104
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad subdatasets
+
+Not only the ``longnow`` subdataset, but also the newly created
+``midterm_project`` subdataset are displayed -- wonderful!
+
+After the last lecture, you naturally want your dataset to follow the YODA
+principles. As a start, you use the ``cfg_yoda`` procedure to help you
+structure the dataset [#f1]_.
+
+.. runrecord:: _examples/DL-101-130-105
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ cd midterm_project
+   $ datalad run-procedure cfg_yoda
```

Comment:
I would make sense to me to to this simultaneously with `create`, but it may be just my taste.